Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.17.6-bullseye
+
+ENV UID=1000
+ENV GID=1000
+ENV GOXYGEN_DOCKER=true
+
+WORKDIR /app
+COPY . .
+
+# Without this line, the docker image will not be able to run and produce error: 
```

Comment:
Let's remove these comments. I guess it is fine considering the app is supposed to generate a Git repository.